content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -195,19 +195,19 @@ datasource db {
 }
 ```
 
-In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+You can use the `env()` function in the following places:
+
+- A datasource url
+- Generator binary targets
+
+In version 2.7.0 and later, you can expand any variables stored in the `.env` file. Use the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products such as Heroku):
```

Comment:
Opening `(` is missing (as it was before as well).

Reply:
```suggestion
In version 2.7.0 and later, you can expand any variables stored in the `.env` file. Use the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). (This is useful when dealing with PaaS or similar products such as Heroku):
```

Reply:
I suggest that we lose the brackets around the third sentence here.